README.md

```diff
@@ -1,2 +1,23 @@
-# IBM App Configuration samples
-Sample applications for App Configuration service
+# Samples for IBM Cloud App Configuration
+App Configuration is a centralized feature management and configuration service for use with web and mobile applications, micro-services, and distributed environments.
+
+These samples are aimed at providing a way to quickly configure App Configuration service for running demos and exploring functionality. They are not designed for production use. All samples are relatively small so that they can be re-used easily and integrated into a larger use-case.
+
+### Apps
+- [Node.js microservice](nodejs)
+- [Android app (Java)](android-java)
+- [Android app (Kotlin)](android-kotlin)
+
+### Custom resource based on Razee
+- [Razee](razee)
+
+## Additional Resources
+
+- [IBM Cloud App Configuration](https://cloud.ibm.com/catalog/services/app-configuration#about)
+- For questions/comments join us on Slack:<br>
```

Comment:
Lets not point external users to the slack channel since it is not public

Reply:
Ok. updated